applications/GeoMechanicsApplication/custom_conditions/micro_climate_constants.h

```diff
@@ -0,0 +1,34 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#pragma once
+
+namespace Kratos::MicroClimateConstants
+{
+
+constexpr static auto AirDensity = 1.18;
```

Comment:
I now see (next to WPKs comments) that there might be a better way to sort these constants